processing/sql/dml/002_prg_pa_update.sql

```diff
@@ -0,0 +1,3 @@
+-- wczyszczenie pol w tabeli pa
```

Comment:
to chyba jest zbędne, bo w poprzednim kroku tworzymy tabelę od nowa i te pola i tak będą miały nulle?

Reply:
masz racje. ten update jest niepotrzebny. To takie zboczenie zawodowe. Uczyli mnie, ze przed wypełnianiem tabeli należy zawsze upewnić sie te tabela jest pusta.

Reply:
Spoko, można to zostawić :) Ja w drugą stronę patrzę, bo update-y na Postgresie są dość kosztowne przez implementację MVCC, ale tutaj korzystamy z tabeli unlogged (przynajmniej na tym etapie procesu) to właściwie nie powinno mieć znaczenia większego.